geode-docs/getting_started/intro_to_clients.html.md.erb

```diff
@@ -0,0 +1,357 @@
+
+      System.out.println(cache.getDefaultPool().getLocators());
+
+    cache.close();
+ }
+}
+```
+
+The information printed out should match the host and port of your <%=vars.product_name_long%> instance locators and should resemble
+
+```
+[/127.0.0.1:10334]
+```
+
+**Simple Put and Get with <%=vars.product_name_long%> Java client**
+ 
+```
+public static void main(String[] args) {
+ClientCache cache = new ClientCacheFactory().addPoolLocator("127.0.0.1", 10334).create();
+
+ Region<String, String>
+     helloWorldRegion =
+     cache.<String, String>createClientRegionFactory(ClientRegionShortcut.PROXY).create("helloWorld");
+
+ helloWorldRegion.put("1", "HelloWorldValue");
+ String value1 = helloWorldRegion.get("1");
+ System.out.println(value1);
+
+ cache.close();
+}
+```
+
+Build and run the application.  This puts the key ‘1’ with a value of ‘HelloWorldValue’ into the
+‘helloWorld’ region, then retrieves and prints the value of key ‘1’.
+
+**Additional Resources**
+
+- [Apache Geode Examples GitHub Repo](https://github.com/apache/geode-examples)
+- [Apache Geode Javadocs](https://geode.apache.org/releases/latest/javadoc/index.html)
+
+## <a id="client-intro-spring-boot"></a>Spring Boot Client For Apache Geode
+
+Spring Boot for Apache Geode provides a powerful abstraction that simplifies the developer
+experience when using Spring Boot and Apache Geode.  The best way to get started with Spring Boot
+for Apache Geode, is by creating a project using [Spring&trade; initializr](https://start.spring.io/).
+
+Add the ‘Spring for Apache Geode’ dependency and then select either ‘Generate’, which creates a zip
+file to import into your IDE or ‘Explore’, which opens a build file based on your Project selection
+(Maven or Gradle)
+
+**Maven Dependencies**
+
+Your Maven pom.xml file should include something similar to 
+
+```
+<properties>
+   ...
+   <spring-geode.version>1.4.3</spring-geode.version>
+ </properties>
+ <dependencies>
+   <dependency>
+     <groupId>org.springframework.geode</groupId>
+     <artifactId>spring-geode-starter</artifactId>
+ </dependency>
+…
+</dependencies>
+
+ <dependencyManagement>
+   <dependencies>
+     <dependency>
+       <groupId>org.springframework.geode</groupId>
+       <artifactId>spring-geode-bom</artifactId>
+       <version>${spring-geode.version}</version>
+       <type>pom</type>
+       <scope>import</scope>
+     </dependency>
+   </dependencies>
+ </dependencyManagement>
+```
+
+**Gradle Dependencies**
+
+Your Gradle gradle.build file should include something similar to 
+
+```
+ext {
+ set('springGeodeVersion', "1.4.3")
+}
+
+dependencies {
+ implementation 'org.springframework.geode:spring-geode-starter'
+ testImplementation 'org.springframework.boot:spring-boot-starter-test'
+}
+dependencyManagement {
+ imports {
+   mavenBom "org.springframework.geode:spring-geode-bom:${springGeodeVersion}"
+ }
+}
+```
+
+1. Connect to Geode from your application
```

Comment:
I would advocate that instead of this being 1. Connect to Geode from your application, that it instead be:
** Connect to Geode from Your Application**

The way it's styled now makes it appear as though it goes in order with the "Gradle dependencies" section.  

So it would be 
1. Open your web browser to Spring Initializr...

Reply:
Corrected.